csharp/ql/src/experimental/Security Features/backdoor/DangerousNativeFunctionCall.qhelp

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>This query finds native calls to external functions that are often used in creating backdoors or are generally attributed to unsafe code practices.</p>
```

Comment:
I think it's worth mentioning Solorigate as an example in the help text, and include some examples of a few of these external functions.

Reply:
I am adding the same comment than for ProcessNameToHashTaintFlow

Reply:
I see you've added the text to the qldoc in the `.ql` file — thanks! Would you mind adding a reference to Solorigate to this `.qhelp` file too? I understand the query is a more generic one than the others (in the `Solorigate` directory), but it'd be good to tell users that Solorigate is an example of a backdoor.

Reply:
I added a similar text than we used for the other rules to all the backdoor queries. Please let me know if you would like me to change anything in the text.